apps/mobile-wallet/src/components/layout/Screen.tsx

```diff
@@ -29,23 +30,36 @@ export interface ScreenProps extends ViewProps {
   headerOptions?: BaseHeaderOptions & {
     type?: 'default' | 'stack'
   }
+  safeAreaPadding?: boolean
+  usesKeyboard?: boolean
 }
 
-const Screen = ({ children, headerOptions, ...props }: ScreenProps) => {
+const Screen = ({ children, headerOptions, safeAreaPadding, usesKeyboard, ...props }: ScreenProps) => {
```

Comment:
Added the same functionalities as `ScrollScreen`. We used to use scroll screens when not actually needed. I've replaced some of them with simple non-scrollable screens.